src/Catty/FormulaEditor/ViewController/FormulaEditorViewController+Section.swift

```diff
@@ -22,51 +22,6 @@
 
 extension FormulaEditorViewController {
```

Comment:
Since this extension is not directly related to sections, my suggestion would be to rename it to `FormulaEditorViewControllerInputExtension` or similar. What do you think?

Reply:
Yes, I have renamed that file to `FormulaEditorViewControllerInputExtension`.